webapp/src/pages/Settings.tsx

```diff
@@ -783,6 +793,69 @@ const Settings: React.FC<Props> = ({ onClose }) => {
     </FormGroup>
   );
 
+  const getCommonFieldsConfigSection = () => (
+    <Box marginY={1.5}>
```

Comment:
I need another option. @gabegma, do you think we should add a "General" sub-title to the new section even though there are no other sub-sections? That way it would persist the same "structure" as the other sections.
![image](https://user-images.githubusercontent.com/8386369/226797695-6cfd9049-7950-45d1-b77c-0253d2f81c8e.png)

Reply:
Oh interesting! I actually don't think it looks so odd like that. I'm not sure we need to add "General". What do you think?

Reply:
I good either way. Let's keep it like that.
The only thing I'd do is adjust the spacing, to give the result shown in my previous screenshot.
```suggestion
    <Box marginTop={2}>
```
It doesn't look as sharp (not centered) as it is now:
![image](https://user-images.githubusercontent.com/8386369/226802131-9b1257e4-c2b5-4b9b-9b68-5c1c4dbf7dd6.png)